ExampleSwift/ExampleSwift/Sources/API/Create Customer/CreateCustomerRequest.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  CreateCustomerRequest.swift
+//  ExampleSwift
+//
+//  Created by Jarrod Robins on 15/5/2023.
+//  Copyright © 2023 Airwallex. All rights reserved.
+//
+
+import Foundation
+
+struct CreateCustomerRequest: Encodable {
+    
+    enum CodingKeys: String, CodingKey {
```

Comment:
Can we just use [JSONDecoder.KeyDecodingStrategy.convertFromSnakeCase](https://developer.apple.com/documentation/foundation/jsondecoder/keydecodingstrategy/convertfromsnakecase) in the decoding rather than specifying coding keys throughout?

Reply:
never mind - I see the IDs now